client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -19,9 +19,25 @@ import EditorMediaModalDetailPreviewAudio from './detail-preview-audio';
 import EditorMediaModalDetailPreviewDocument from './detail-preview-document';
 import Button from 'components/button';
 import { userCan, isJetpack } from 'lib/site/utils';
+import versionCompare from 'lib/version-compare';
 import MediaUtils, { isItemBeingUploaded } from 'lib/media/utils';
 import config from 'config';
 
+/**
+ * This function returns true if the site is a Jetpack site
+ * and its version is greater than the minimum version
+ * to be able to edit media files.
+ *
+ * @param  {object} site - current site
+ * @return {boolean} `true` is it's a valid Jetpack site
+ */
+const allowForJetpackSite = site => (
```

Comment:
Maybe we should update this method to just check if the image editor is enabled for all cases?

Right now the logic is being added incorrectly with the other cases, and so the image editor is not visible to normal wpcom sites.